backend/stakepoold/rpcclient.go

```diff
@@ -192,27 +192,20 @@ func walletFetchUserTickets(ctx *appContext) map[string]UserTickets {
 			continue
 		}
 		if spui != nil && len(spui.Tickets) > 0 {
-			tickets := make([]*chainhash.Hash, 0)
 			for _, ticket := range spui.Tickets {
 				switch ticket.Status {
 				case "live":
 					hash, err := chainhash.NewHashFromStr(ticket.Ticket)
 					if err != nil {
 						log.Infof("invalid ticket %v", err)
-					} else {
-						tickets = append(tickets, hash)
+						continue
 					}
+
+					userTickets[hash.String()] = ctx.userVotingConfig[msa].MultiSigAddress
```

Comment:
You just did `hash, err := chainhash.NewHashFromStr(ticket.Ticket)`.  So why do `userTickets[hash.String()]` when you already have the string?  Isn't `userTickets[ticket.Ticket]` correct?